src/wrappers/themis/wasm/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wasm-themis",
-  "version": "0.14.0",
+  "version": "0.14.1",
```

Comment:
This should become 0.14.5, I believe, if you're going to make a patch release.

Reply:
i agree, let's make `0.14.5`